app/controllers/api_applications_controller.rb

```diff
@@ -20,15 +20,11 @@ def edit
   def create
     @api_application = current_user.api_applications.new(params[:api_application])
 
-    respond_to do |format|
-      if @api_application.save
-        format.html do
-          redirect_to @api_application,
-                      notice: 'Application was successfully created.'
-        end
-      else
-        format.html { render :new }
-      end
+    if @api_application.save
+      redirect_to edit_api_application_path(@api_application),
+                  notice: 'Application was successfully created.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.